### PR TITLE
Allows to compute the trap spaces intersecting a given subspace

### PR DESCRIPTION
**Pull request by Lauracf**

### CODE REVIEW

**Comment by Lauracf** [comment]
What went wrong?

**Comment by hklarner** [comment]
was caused by an update to a package in the ubuntu that runs our tests. `libreadline7` was replaced by `libreadline8`. I had to update the link. `nusmv-a` still needs `libreadline6`. :roll_eyes: 

- update your PR by pulling the current develop into your branch

**Comment by Lauracf** [comment]
> was caused by an update to a package in the ubuntu that runs our tests. `libreadline7` was replaced by `libreadline8`. I had to update the link. `nusmv-a` still needs `libreadline6`. roll_eyes
> 
>     * update your PR by pulling the current develop into your branch

I tried to pull the current develop and it told me that it is already up to date.

**Comment by hklarner** [comment]
hm. sounds like you are talking about `origin` not `upstream`. try this:

```
git fetch upstream
git merge upstream/develop develop
```

post the output of `git log` if it doesn't work.

finally, here is the commit on `develop` that you need
- https://github.com/hklarner/PyBoolNet/commit/0f12fa7221f4879a7a34daae7c963a30e879b217

**Comment by Lauracf** [comment]
Thanks, I forgot to do `git fetch upstream`

**Comment by hklarner** [comment]
@Lauracf in case you were not notified about my comments I think this notifies you.

**Comment by Lauracf** [comment]
I already did what you said. Is there something else missing?

**Comment by hklarner** [comment]
Your last commit is 15, March.

**Comment by Lauracf** [comment]
Yes, that day I did the merge and the automatic checks worked

**Comment by hklarner** [comment]
oh but I started a review above

File: PyBoolNet/PrimeImplicants.py [hunk 1]
```diff
@@ -680,18 +680,22 @@ def input_combinations(Primes, Format="dict"):
 
 
 
-def active_primes(Primes, State):
+def active_primes(Primes, Subspace):
```
[the remark this answers is not in the document]
Comment (hklarner) [reply PyBoolNet/PrimeImplicants.py:683]: Hey what happened?

**Pull request merged**